pkg/adhoc/server/convert.go

```diff
@@ -118,8 +110,7 @@ func DiffV1(name string, base, diff *flamebearer.FlamebearerProfile, maxNodes in
 		name = n
 	}
 
-	fb = flamebearer.NewCombinedProfile(name, out, bOut, dOut, maxNodes)
-	return fb, nil
+	return flamebearer.NewCombinedProfile(name, bOut, dOut, maxNodes)
```

Comment:
Note that `bOut`, `dOut` don't have metadata currently, they should be completed with metadata from `base` and `diff`.

Reply:
good catch, thanks!